bigbluebutton-html5/imports/ui/components/tooltip/component.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Tippy from 'tippy.js';
+import _ from 'lodash';
+import cx from 'classnames';
+
+const propTypes = {
+  title: PropTypes.string.isRequired,
+  position: PropTypes.oneOf(['bottom']),
+  children: PropTypes.element.isRequired,
+  dynamicTitle: PropTypes.bool
```

Comment:
This propType is causing a console warning.
![console_error](https://user-images.githubusercontent.com/22058534/33846509-2efff10c-de75-11e7-9581-0cf3e898c988.png)